AMDirT/cli.py

```diff
@@ -0,0 +1,259 @@
+import click
+from AMDirT import __version__
+
+from AMDirT.validate import run_validation
+from AMDirT.viewer import run_app
+from AMDirT.convert import run_convert
+from AMDirT.core import get_json_path
+from AMDirT.autofill import run_autofill
+from AMDirT.merge import merge_new_df
+from json import load
+
+
+def get_table_list():
+    json_path = get_json_path()
+    with open(json_path, "r") as f:
+        table_list = load(f)
+        return list(table_list["samples"].keys())
+
+
+@click.group()
+@click.version_option(__version__)
+@click.pass_context
+@click.option("--verbose", is_flag=True, help="Verbose mode")
+def cli(ctx, verbose, no_args_is_help=True, **kwargs):
+    """\b
+    AMDirT: Performs validity check of ancientMetagenomeDir datasets
+    Authors: AMDirT development team and the SPAAM community
+    Homepage & Documentation: https://github.com/SPAAM-community/AMDirT
+    \b
+    """
+    ctx.ensure_object(dict)
+    ctx.obj["verbose"] = verbose
+    pass
+
+
+####################
+# Validation  tool #
+####################
+
+
+@cli.command()
+@click.argument("dataset", type=click.Path(exists=True))
+@click.argument("schema", type=click.Path(exists=True))
+@click.option("-s", "--schema_check", is_flag=True, help="Turn on schema checking.")
+@click.option(
+    "-d", "--line_dup", is_flag=True, help="Turn on line duplicate line checking."
+)
+@click.option(
+    "-c", "--columns", is_flag=True, help="Turn on column presence/absence checking."
+)
+@click.option("-i", "--doi", is_flag=True, help="Turn on DOI duplicate checking.")
+@click.option(
+    "--multi_values",
+    multiple=True,
+    help="Check multi-values column for duplicate values.",
+)
+@click.option(
+    "-a",
+    "--online_archive",
+    is_flag=True,
+    help="Turn on ENA accession validation",
+)
+@click.option(
+    "--remote",
+    type=click.Path(),
+    default=None,
+    help="[Optional] Path/URL to remote reference sample table for archive accession validation",
+)
+@click.option("-m", "--markdown", is_flag=True, help="Output is in markdown format")
+@click.pass_context
+def validate(ctx, no_args_is_help=True, **kwargs):
+    """\b
+    Run validity check of ancientMetagenomeDir datasets
```

Comment:
```suggestion
    Run validity check of AncientMetagenomeDir datasets
```

I woudl change everywhere